web/book/src/language-features/README.md

```diff
@@ -3,16 +3,20 @@
-| ---------------- | ----------------------------------------------------------------------- |
-| Coalesce         | [Handle potentially NULL values](./coalesce.md)                         |
-| Dates & times    | [Handle dates and times](./dates-and-times.md)                          |
-| Distinct         | [Equivalent of SQL `DISTINCT`](./distinct.md)                           |
-| Null handling    | [Handle `NULL` values](./null.md)                                       |
-| Ranges           | [Syntax for all forms of ranges](./ranges.md)                           |
-| Regex            | [Handle regular expressions](./regex.md)                                |
-| Strings          | [Rules for creating strings](./strings.md)                              |
-| S-strings        | [Insert SQL directly into a query with an S-string](./s-strings.md)     |
-| F-strings        | [Combine several column's data with F-strings](./f-strings.md)          |
-| Case             | [Create a new column based on the contents of other columns](./case.md) |
-| Target & Version | [Specify a target SQL engine and PRQL version](./target.md)             |
+<!-- we're linking to README.md files as index.html to work around https://github.com/rust-lang/mdBook/issues/984  by using "markdown_link_check_disable"-->
+
+| Feature                                               | Purpose                                                                 |
+| ----------------------------------------------------- | ----------------------------------------------------------------------- |
+| Case                                                  | [Create a new column based on the contents of other columns](./case.md) |
+| Coalesce                                              | [Handle potentially NULL values](./coalesce.md)                         |
+| Dates & times                                         | [Handle dates and times](./dates-and-times.md)                          |
+| Distinct                                              | [Equivalent of SQL `DISTINCT`](./distinct.md)                           |
+| Relation Literals                                     | [Define a table](./relation-literals.md)                                |
+| Null handling                                         | [Handle `NULL` values](./null.md)                                       |
+| Ranges                                                | [Syntax for all forms of ranges](./ranges.md)                           |
+| Regex                                                 | [Handle regular expressions](./regex.md)                                |
+| <!-- markdown-link-check-disable --> Standard Library | [Common functions in SQL](./standard-library/index.html)                |
+| <!-- markdown-link-check-enable--> Strings            | [Rules for creating strings](./strings.md)                              |
```

Comment:
Having these comments in the table is rather ugly. I think it should be possible to put the link definition after the table i.e. having just `[Common functions in SQL]` in the table and then after the table declaring:

```
[Common functions in SQL]: ./standard-library/index.html
```

Reply:
I am not fond of the comments either, but I'm not sure how to eliminate them without removing the link-check for the entire table. 

Or am I missing something? I would entertain a different suggestion. Thanks.

Reply:
Yes ... you're missing what I said. Markdown links can be broken up like:

```
[test]

[test]: foo.md
```

Reply:
Hah! You're right. Thanks for reminding me that Markdown has this facility.